src/AggregateFunctions/AggregateFunctionGroupBitmap.cpp

```diff
@@ -72,9 +71,11 @@ namespace
 
         const DataTypeAggregateFunction & datatype_aggfunc = dynamic_cast<const DataTypeAggregateFunction &>(*argument_type_ptr);
         AggregateFunctionPtr aggfunc = datatype_aggfunc.getFunction();
-        argument_type_ptr = aggfunc->getArgumentTypes()[0];
+        DataTypePtr nested_argument_type_ptr = aggfunc->getArgumentTypes()[0];
```

Comment:
Consider to leave comment related to this.

Reply:
Ok.